app/src/main/java/com/example/fizzbuzz/fizzbuzz.kt

```diff
@@ -1,5 +1,70 @@
 package com.example.fizzbuzz
 
+fun addWord(words: MutableList<String>, word: String) {
+    words.add(word)
+}
+
+fun setWordsToBong(words: MutableList<String>) {
+    words.clear()
+    words.add("Bong")
+}
+
+fun insertFezz(words: MutableList<String>) {
```

Comment:
Here too, we could utilise some Kotlin higher-order functions to make this a little more concise and readable. 
e.g. something kinda like
```
fun insertFezz(words: MutableList<String>) {
    val index = words.indexOfFirst { it.startsWith('B') }
    if (index != -1) {
        words.add(index, "Fezz")
    } else {
        words.add("Fezz")
    }
}
```